piptools/utils.py

```diff
@@ -48,13 +48,33 @@ def make_install_requirement(name, version, extras, constraint=False):
     )
 
 
+def is_url_requirement(ireq, repository=None):
+    """
+    Finds if a requirement is a URL
+    """
+    if (
```

Comment:
Could you please describe in the comment line why `"files.pythonhosted.org" in ireq.link.url` is not an URL requirement?